### DISCUSSION TITLE
can we have q-notify tag for customizing notifications. ?

### DISCUSSION THREAD

**Discussion by rajeshtva** [quasarframework/quasar#13995] [Ideas / Proposals]
we already have `QDialog` component for dialogs. but i think there should also be a notifications component wrapper, which can be customized according to need  currently we are stuck with very few options which can be customized only by props in `QNotify` api.

Vuetify has implemented it in the `Alert` form. kindly check [this link](https://vuetifyjs.com/en/api/v-alert/). I would prefer customization like this.

**Comment by MilosPaunovic** [comment]
Couldn't you achieve this by setting defaults:[
https://quasar.dev/quasar-plugins/notify#setting-defaults](https://quasar.dev/quasar-plugins/notify#setting-defaults)

**Reply by rajeshtva** [reply]
i use it very much. but it doesn't provide much customization. what if we want completely different ui than provided by default. for example is [this kind](https://lh3.googleusercontent.com/LPEyZ21OejaGsaZXGfWGiwpribS0rOu7rP2CSOuAVHMX83B638uVyHnD9P9BAcO4VmQTO7fxkZP-Wfc9FibmV4MI73fxJt8-Qn6zxw=w1064-v0) of exact design possible with current notification system? or something like [this vuetify component](https://vuetifyjs.com/en/components/alerts/#colored-border). 

do i need to use dialog for creating these kind of ui or some tweaking in notify component is enough?.

**Reply by MilosPaunovic** [reply]
Maybe this can help you: [https://quasar.dev/quasar-plugins/notify#using-html](https://quasar.dev/quasar-plugins/notify#using-html)
